src/styles/main.scss

```
@@ -12,3 +12,35 @@ body {
 
 @import "_header.scss";
 @import "_catalog.scss";
+
+@media (min-width: 1px) and (max-width: 487px) {
+  .catalog {
+    display: grid;
+    gap: 48px;
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+
+@media (min-width: 488px) and (max-width: 767px) {
+  .catalog {
+    display: grid;
+    gap: 48px;
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) and (max-width: 1023px) {
+  .catalog {
+    display: grid;
+    gap: 48px;
+    grid-template-columns: repeat(3, 200px);
```

```suggestion
    grid-template-columns: repeat(3, 200px);
```
У всіх медіа-запитах дублюються стилі. В тебе міняється лише grid-template-columns, от його лише і варто прописувати